themes/courses.labster.com/lms/templates/ccx/enrollment.html

```diff
@@ -97,9 +97,8 @@ <h2 class="hd hd-2">${_("Batch Enrollment")}</h2>
             <!-- Start: Added by Labster -->
             <!-- Add enrollment status -->
             %for member in ccx_enrolls:
-            <% username = member.email.split('@')[0].lower() %>
             <tr>
-              <td>${username}</td>
+              <td>${member.email}</td>
```

Comment:
does it make sense to duplicate it? can we leave it empty?

Reply:
changed